defaults/main.yml

```diff
@@ -5,6 +5,12 @@
 # options on different hosts or server directories.
 galaxy_toolshed_manage_static_setup: yes
 galaxy_toolshed_manage_database: yes
+# When creating the shed use bootstrapping approach against an existing
+# shed instead of creating from scratch.
+galaxy_toolshed_bootstrap: no
+galaxy_toolshed_bootstrap_username: devteam
```

Comment:
should this be consistent with ~/.planemo.yml? (planemo/planemo/planemo@test.com)